### PR TITLE
Add to flash scripts

### PR DESCRIPTION
**Pull request by bthaile**
give ability for user to connect using different account.
always use contract owner to change timestamp
started initial-report but need assistance to get SDK ready to get marketInfo

### CODE REVIEW

File: packages/augur-sdk/src/utils.ts [hunk 1]
```diff
@@ -124,3 +127,37 @@ export function calculatePayoutNumeratorsValue(
...
+  numTicks,
+  numOutcomes,
+  marketType,
+  outcome
+): BigNumber [] {
+  const payoutNumerators = Array(numOutcomes).fill(new BigNumber(0));
+  const isScalar = marketType === MarketTypeName.Scalar;
+
+  if (isScalar) {
+    // selectedOutcome must be a BN as string
```
Comment (robert-e-davidson3) [inline packages/augur-sdk/src/utils.ts]: `selectedOutcome` doesn't seem to be a thing

File: packages/augur-tools/src/constants.ts [hunk 2]
```diff
@@ -13,9 +13,44 @@ export const ACCOUNTS: Account[] = [
     publicKey: "0x913dA4198E6bE1D5f5E4a40D0667f70C0B5430Eb",
```
Comment (robert-e-davidson3) [inline packages/augur-tools/src/constants.ts:13]: IMO the first account should have a comment indicating that it's considered to be the contract owner.

File: packages/augur-tools/src/flash/scripts.ts [hunk 3]
```diff
@@ -1,28 +1,42 @@
...
     options: [
       {
-        name: "network",
+        name: 'account',
+        abbr: 'a',
+        description: `account address to connect with, if no address provided contract owner is used`,
+      },
+      {
+        name: 'network',
+        abbr: 'n',
```
Comment (robert-e-davidson3) [inline packages/augur-tools/src/flash/scripts.ts:32]: I kind of love that you abbreviated abbreviation.

File: packages/augur-tools/src/flash/scripts.ts [hunk 4]
```diff
@@ -155,105 +178,191 @@ export function addScripts(flash: FlashSession) {
...
       },
     ],
     async call(this: FlashSession, args: FlashArguments) {
       if (this.noProvider()) return;
-      const user = await this.ensureUser();
+      const user = await this.contractOwner();
 
       const countString = args.count as string;
       let unit = countString[countString.length - 1];
+      this.log(`unit ${unit}`);
```
Comment (robert-e-davidson3) [inline packages/augur-tools/src/flash/scripts.ts]: Is this debug code or intentional?

File: packages/augur-sdk/src/utils.ts (hunk 1 above)
Comment (bthaile) [reply packages/augur-sdk/src/utils.ts]: I'll fix that

**Review by robert-e-davidson3 — APPROVED** [review]
LGTM

File: packages/augur-tools/src/flash/scripts.ts (hunk 4 above)
Comment (bthaile) [reply packages/augur-tools/src/flash/scripts.ts]: remove it

**Pull request merged**